middle.mkd

```diff
@@ -331,14 +331,17 @@ height in meters above or below the WGS 84 reference ellipsoid. In the
 absence of elevation values, applications sensitive to height or depth
 SHOULD interpret positions as being at local ground or sea level.
 
-Other coordinate reference systems, including ones described by CRS
-objects of the kind defined in [GJ2008] are NOT RECOMMENDED. GeoJSON
+Other coordinate reference systems are NOT RECOMMENDED. GeoJSON
 processing software SHALL NOT be expected to have access to coordinate
-reference systems databases. Applications requiring a CRS other than the
-default MUST assume all responsibility for CRS identification,
-coordinate accuracy, and interpretation of missing elevation values.
-Furthermore, GeoJSON coordinates MUST NOT under any circumstances use
-latitude, longitude order.
+reference system databases or be expected to parse and interpret the CRS
+objects described in [GJ2008]. Furthermore, to prevent coordinate order
+confusion for parsers that do not have a CRS database, GeoJSON
+coordinates MUST NOT under any circumstances use latitude, longitude
+order. For example, GeoJSON data using a coordinate reference system
```

Comment:
I think that this is too much.  If someone is using a different CRS in GeoJSON, then they are taking their chances already.  Better perhaps to gently recommend that expressing coordinates in latitude, longitude order risks compounding errors where a recipient of GeoJSON does not expect a different CRS.